packages/zowe-explorer/src/shared/HistoryView.ts

```diff
@@ -95,9 +97,14 @@ export class HistoryView extends WebView {
             fileHistory: treeProvider.getFileHistory(),
             dsTemplates: type === "ds" ? (treeProvider as DatasetTree).getDsTemplates() : undefined,
             favorites: treeProvider.getFavorites(),
+            encodingHistory: type === "uss" || type === "ds" ? this.fetchEncodingHistory() : [],
         };
     }
 
+    private fetchEncodingHistory(): string[] {
+        return ZoweLocalStorage.getValue<string[]>("zowe.encodingHistory") ?? [];
```

Comment:
I believe we should move the `"zowe.encodingHistory"` into the `LocalStorageKey` enum.

---

Also, thanks for doing `?? []` at the end.
This is very similar to how [this issue](#2770) below was fixed in [this PR](#2783)

Reply:
Hi, Where is this enum located?

Reply:
Oh, my apologies, I mixed up my PR reviews.
it was added only in the `next` branch in [this PR](https://github.com/zowe/zowe-explorer-vscode/pull/2788)
**_Source:_** [packages/zowe-explorer/src/utils/ZoweLocalStorage.ts](https://github.com/zowe/zowe-explorer-vscode/blob/f3cebbd8f00ad84173caa2ff67bb9e342f8fbc81/packages/zowe-explorer/src/utils/ZoweLocalStorage.ts)


No need to create a new enum just for these.